nemo/collections/nlp/models/language_modeling/megatron_finetune_model.py

```diff
@@ -200,25 +199,6 @@ def on_train_epoch_start(self) -> None:
         self.on_validation_epoch_end()
         return super().on_train_epoch_start()
 
-    def training_step(self, batch, batch_idx):
```

Comment:
why are we removing this?

Reply:
Ah thanks @aklife97 

Because we do not have access to the batch we're not able to run this check.

I think for now what we can do is replace this with an assertion to ensure that `drop_last=True` is always true. What do you think?

Reply:
I think we're good without an assert here, since we anyways don't support drop_last as False at train time. So looks good!